v4.4.3

jackyaz opened the pull request:
CHANGED: If InfluxDB port is set as 443, use https

jackyaz committed:
Version bump

jackyaz committed:
Assume https if port is 443

jackyaz committed:
Download README and LICENSE if they don't exist locally

waeltken commented:
Hey @jackyaz, I would love to try this!

waeltken commented:
I've tried the https implementation with InfluxDB v2.3.0 and was able to make it work by fiddling with the curl line as shown here: #60

Martinski4GitHub committed:
Update connmonstats_www.asp

Made changes to comply with JS dependencies when loading the WebGUI page on the 3006.102.1 F/W version.

Martinski4GitHub committed:
Update README.md

Updated version string & date.

Martinski4GitHub committed:
Update connmon.sh

Updated version string.

Martinski4GitHub committed:
Updated ASP file.

jackyaz committed:
Merge pull request #61 from Martinski4GitHub/develop

Changes to support 3006.102.1 F/W

Martinski4GitHub committed:
Update README.md

Corrected next version string.

Martinski4GitHub committed:
Update connmon.sh

Corrected next version string.

Martinski4GitHub committed:
Fixed "Reset Database" functionality

Fixed the "Reset Database" functionality on the CLI menu where the database file was correctly reset but the result was not reflected on the webGUI page where "old" entries were still shown as if the database had not been reset.

Martinski4GitHub committed:
Numerical comparison operator.

Updated numerical comparison operator.

jackyaz committed:
Merge pull request #62 from Martinski4GitHub/develop

Fixed "Reset Database" functionality

Martinski4GitHub committed:
Updated for 3.0.3 Release

Added latest fixes for the 3.0.3 release version.

jackyaz committed:
Merge pull request #63 from Martinski4GitHub/develop

Updated for 3.0.3 Release

Martinski4GitHub committed:
Code improvement.

Removed/modified code related to "var $j = jQuery.noConflict();" which is now considered obsolete.

Martinski4GitHub committed:
Fixed issue reported by Codacy

Fixed an issue reported by the Codacy Static Analysis check:
Function "buildLastXTable()" was used before it was defined.

Martinski4GitHub committed:
Functionality Improvements

1) IMPROVED: Modified all SQLite3 calls to capture and log errors in the system log.

2) IMPROVED: Modified SQLite3 configuration parameters to improve the trimming of records from the database and then perform "garbage collection" of deleted entries to reclaim unused space & avoid excessive fragmentation.

3) IMPROVED: Modified SQLite3 configuration parameters to improve the processing database records.

4) IMPROVED: Modified code to set the corresponding priority level of log entries when calling the built-in logger utility.

5) IMPROVED: Modified the startup call made in the post-mount script to check if the USB-disk partition passed as argument has Entware installed.

6) IMPROVED: Added code to show the current database file size information on the CLI menu and the webGUI page.

7) IMPROVED: Added code to show the "JFFS Available" space information for the "Data Storage Location" option on the CLI menu and the webGUI page.
   This allows users to see if/when the "JFFS Available" space is getting too low to store the database and additional supporting files (CSV, HMT, configs) when the "Data Storage Location" is set to JFFS.

8) IMPROVED: Miscellaneous coding improvements & fine-tuning.

Martinski4GitHub committed:
Update connmonstats_www.js

Fixing Codacy Static Code Analysis check errors.

Martinski4GitHub committed:
Update connmonstats_www.js

Fixing Codacy Static Code Analysis check errors.

Martinski4GitHub committed:
Update connmonstats_www.js

Fixing Codacy Static Code Analysis check errors.

Martinski4GitHub committed:
Update connmonstats_www.js

Fixing Codacy Static Code Analysis check errors.

Martinski4GitHub committed:
Update connmonstats_www.js

Fixing Codacy Static Code Analysis check error.

Martinski4GitHub committed:
Update connmonstats_www.asp

Fine-tuning code.

Martinski4GitHub committed:
Update CHANGELOG.md

Updated.

Martinski4GitHub committed:
Code Improvements.

Fine-tuning code.

Martinski4GitHub committed:
Update connmon.sh

Fine-tuning code.

Martinski4GitHub committed:
Update connmon.sh

Fine-tuning.

Martinski4GitHub committed:
Code Improvements

Some code improvements & fine-tuning, most related to the curl command calls.

Martinski4GitHub committed:
Update connmon.sh

More fine-tuning.

jackyaz committed:
Merge pull request #64 from Martinski4GitHub/develop

Code improvements

Martinski4GitHub committed:
Code Improvements

Improvements & fine-tuning code.

Martinski4GitHub committed:
Update connmon.sh

Fixed the database journaling method so that SQLite3 commands work when the database in stored in JFFS; otherwise, some errors are generated.

Martinski4GitHub committed:
Fine-tuning code.

Martinski4GitHub committed:
Code Improvements

Some code improvements & fine-tuning.

Martinski4GitHub committed:
Update connmon.sh

Indentation changes.

Martinski4GitHub committed:
Fixed JFFS Space Calculation

Fixed calculation of JFFS Available vs Required Space.

jackyaz committed:
Merge pull request #65 from Martinski4GitHub/develop

Fixed Database Journaling Method
